src/core/shape/2d_primitives.js

```diff
@@ -230,9 +230,9 @@ p5.prototype.ellipse = function(x, y, w, h, detailX) {
  * white circle with black outline in mid of canvas that is 55x55.
  */
 p5.prototype.circle = function() {
-  var args = Array.prototype.slice.call(arguments, 0, 2);
-  args.push(2 * arguments[2]);
-  args.push(2 * arguments[2]);
+  let args = Array.prototype.slice.call(arguments, 0, 2);
```

Comment:
`let` is not yet permitted in the source of p5, please use `var` for now

Reply:
thanks! just fixed it.